pandas/core/base.py

```diff
@@ -777,6 +777,118 @@ def base(self):
                       FutureWarning, stacklevel=2)
         return self.values.base
 
+    @property
+    def array(self):
+        # type: () -> Union[np.ndarray, ExtensionArray]
+        """The actual Array backing this Series or Index.
```

Comment:
```suggestion
        """The actual array backing this Series or Index.
```

Reply:
And the summary sentence needs to start on the next line (to follow our docstring guidelines)

(same for `to_numpy`)